dataFields/ABFieldNumberCore.js

```diff
@@ -276,6 +276,7 @@ module.exports = class ABFieldNumberCore extends ABField {
 
    format(rowData) {
       if (
+	 typeof rowData == "undefined" ||
```

Comment:
Would it be great if it can check both `undefined` and `null`

`rowData == null` ?

Reply:
Good point...it is actually null and I was surprised when I did typeof it came back as "undefined". I think just putting `rowData == null` should be enough then.